src/it/scala/net/snowflake/spark/snowflake/IntegrationEnv.scala

```diff
@@ -54,12 +59,57 @@ trait IntegrationEnv
 
   protected val DEFAULT_LOG4J_PROPERTY = "src/it/resources/log4j_default.properties"
 
+  // From spark 3.3, log4j2 is used. For spark 3.2 and older versions, log4j is used.
+  protected val USE_LOG4J2_PROPERTIES = true
```

Comment:
`USE_LOG4J2_PROPERTIES` is introduced to release spark connector for spark 3.2 or older versions.
We only need to change it as true for them.